cmd/ore/aliyun/create.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Red Hat
```

Comment:
Looking at other `cmd/ore/$CLOUD/`'s it seems like this file is generally called `create-image.go`, though there isn't total consensus:

```
$ find . -name "create.go"
./openstack/create.go
$ find . -name "create-image.go"
./azure/create-image.go
./gcloud/create-image.go
./do/create-image.go
```